module5/lessons/Week2/WritingGoodDocumentation.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: page
+title: Writing Good Documentation
+---
+
+## Learning Goals
+
+* Understand the reasons for writing a solid README
+* Determine what to include in a README
+* Use your README as a tool for your eventual job search
+
+## Vocab
+* `Markdown`: a lightweight markup language for creating formatted text using a plain-text editor; we use this for READMEs
+* `README`: a form of documentation for a project directory
+
+## Introduction
+
+You have all seen READMEs before. Take for example [this](https://github.com/turingschool-examples/Mod4Week3_Assessment) assessment. Github displays the README file for the viewer since it holds the main information for the project. 
+
+## The Why
+
+We often think of READMEs as an afterthought in our projects, but they are SO important! Here are some ways that READMEs matter to you and your eventual job hunt:
+- To an employer, your README is often the first impression of the work you've done
+- Sometimes employers only scan the README and don't even open the project
+- Your README is how someone knows how to access your project (It should be written in a way that non-technical people can understand)
+- The README documents your experience and process to an employer (which is way more important than the project itself!)
+- Looking back at past READMEs is a great way to remind yourself of all you've done and prepare for interviews
+- **Taken cumulatively, your READMEs document your technical and professional growth**
+
+<section class="call-to-action" markdown="1">
+  <h3>Judging READMEs</h3>
+
+  In your breakout rooms, look through the READMEs below. We're going to go for first impressions here, so spend about 1 minute per README. Make some observations about the READMEs and add some stickies to [this Jamboard](https://jamboard.google.com/d/14atFSLLIqkl1Oa-EI5sfvS8EWaFH9xfpnceFP17mCHs/edit?usp=sharing)
+
+  README Links:
+  * [README #1](https://github.com/sertmer/check-yo-self)
+  * [README #2](https://github.com/Dustin-Mikusko/socialsentiment)
+  * [README #3](https://github.com/kaylagordon/ican2_FE)
+  * [README #4](https://github.com/turingschool-examples/overlook-api)
+  * [README #5](https://github.com/maddielaw/rec-me)
+  * [README #6](https://github.com/dcoleman21/Sweater-Weather)
+  * [README #7](https://github.com/me-md/me-md)
+</section>
+
+<section class="answer" markdown="1">
+### Some potential takeaways!

```

Comment:
This heading doesn't appear as an `<h3>`

Reply:
It doesn't in markdown preview, but when I preview the website it looks good. I think it's because we have markdown="1" as an attribute for the dropdown.